nexfil.py

```diff
@@ -97,9 +97,12 @@ def fetch_meta():
 
 def banner():
     banner = r'''
-__   _ _____ _     _ _____ _____ _
-| \  | |____  \___/  |____   |   |
-|  \_| |____ _/   \_ |     __|__ |_____
+███╗   ██╗███████╗██╗  ██╗███████╗██╗██╗     
```

Comment:
this is very generic looking unfortunately

Reply:
lol. all good. i just wanted to make it shiny :) 
fwiw, the one i created i made using https://manytools.org/hacker-tools/ascii-banner/